### PR TITLE
Add: Get before run for go github release

### PR DESCRIPTION
**Pull request by winterhotlatte**
Add get command before run command.

### CODE REVIEW

File: script/pipeline/Deploy-GithubRelease.ps1 [hunk 1]
```diff
@@ -23,11 +23,25 @@ git config user.name $UserName; if (-not $?) { throw }
...
 
 # Retag
 git tag release --force; if (-not $?) { throw }
 git push $RepoPushURL release --force; if (-not $?) { throw }
 
+# Unset config
+git config user.name --unset
+git config user.email --unset
+
+# Stop-Pipeline
```
Comment (yanocv) [inline script/pipeline/Deploy-GithubRelease.ps1:37]: Fix here.

**Review by yanocv — CHANGES_REQUESTED** [review]
Please fix extra unsets.

File: script/pipeline/Deploy-GithubRelease.ps1 (hunk 1 above)
Comment (winterhotlatte) [reply script/pipeline/Deploy-GithubRelease.ps1:37]: Resolved.

**Review by yanocv — APPROVED** [review]
Good job.

**Pull request merged**